src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -14,43 +15,112 @@ title: Setting Up An Alert
 slug: setting-up-an-alert
 order: 1
 ---
-In the Alerts view, you'll see all existing alerts.
+Alerts provide a convenient way for Redash to notify you when your scheduled queries match some arbitrary criteria. They are often used to monitor system statuses, financial goals, and unusual account activity. Through integrations with Zapier and IFTTT they can also trigger business processes like user on-boarding work flows or support tickets.
+
+You can see all of your existing alerts by clicking `Alerts` on the main navigation menu. This will show you all the alerts in your organization by name, user, current state, and created date.
```

Comment:
```suggestion
You can see all of the existing alerts by clicking `Alerts` on the main navigation menu.
```

Unless you think there is a reason to keep the list of fields, I would rather remove it because it can change over time and we will probably never update this list :)

Also changed `your` to `the`, because it's not a personalized list (yet?).

Reply:
Fixed in 54398a012207261eb1a2bf7a93dfb241e0ba13d3